pyproject.toml

```diff
@@ -81,3 +82,24 @@ extend-exclude = '''
 [tool.isort]
 profile = "black"
 skip_glob = ["lib"]
+
+[tool.mypy]
+mypy_path = ["src"]
+packages = ["hdf5plugin"]
+strict = true
+implicit_optional = true
+warn_unused_configs = true
+
+[[tool.mypy.overrides]]
+module = ["hdf5plugin.test"]
+disable_error_code = [
+    "attr-defined",
+    "no-untyped-def",
+]
+
+[[tool.mypy.overrides]]
+module = [
+    "blosc2",
+    "hdf5plugin._config",
```

Comment:
This is generated at compile time, not typed for now.
It would be good to change this to generate a json file for example and load it in a Python object for proper typing